docs/guides/getting-started-java-spring.md

```diff
@@ -0,0 +1,255 @@
+
+```java
+@Component
+public class ChargeCreditCardWorker {
+
+  private final static Logger LOG = LoggerFactory.getLogger(ChargeCreditCardWorker.class);
+```
+
+4. Add the following method and decorate the `chargeCreditCard` method with `@JobWorker`:
+
+```java
+  @JobWorker(type = "charge-credit-card")
+```
+
+5. Specify a `@Variable(name = "totalWithTax") Double totalWithTax` argument into the `chargeCreditCard` method:
+
+```java
+  public Map<String, Double> chargeCreditCard(@Variable(name = "totalWithTax") Double totalWithTax) {
+```
+
+6. Lastly, implement the `chargeCreditCard` method:
+
+```java
+   LOG.info("charging credit card: {}", totalWithTax);
+    return Map.of("amountCharged", totalWithTax);
+```
+
+:::note
+If you want to check your work, visit our [sample repository](https://github.com/camunda/camunda-8-get-started-spring/blob/main/src/main/java/io/camunda/demo/process_payments/ChargeCreditCardWorker.java) with the completed code.
+:::
+
+In your terminal, run `mvn spring-boot:run`, where you should see the `charging credit card` output. In Operate, refresh if needed, and note the payment has executed.
+
+## Step 7: Start a process instance
+
+To kick off your process instance, take the following steps:
+
+1. In `ProcessPaymentsApplication.java`, instantiate a static logger:
+
+```java
+@SpringBootApplication
+
+private static final Logger LOG = LoggerFactory.getLogger(ProcessPaymentsApplication.class);
+```
+
+2. Declare `ZeebeClient` within the `@Autowired` annotation:
+
+```java
+	@Autowired
+	private ZeebeClient zeebeClient;
+
+	public static void main(String[] args) {
+		SpringApplication.run(ProcessPaymentsApplication.class, args);
+	}
+```
+
+3. To convert the application to a `CommandLineRunner`, add `implements CommandLineRunner` to the class declaration `public class ProcessPaymentsApplication implements CommandLineRunner`.
+4. Implement an overriding run method:
+
+```java
+	@Override
+	public void run(final String... args) {
+		var processDefinitionKey = "process-payments"; // or whatever the key is
+		var event = zeebeClient.newCreateInstanceCommand()
+				.bpmnProcessId(processDefinitionKey)
+				.latestVersion()
+				.variables(Map.of("total", 100))
+				.send()
+				.join();
+		LOG.info(String.format("started a process: %d", event.getProcessInstanceKey()));
+	}
+```
+
+:::note
+If you want to check your work, visit our [sample repository](https://github.com/camunda/camunda-8-get-started-spring/blob/main/src/main/java/io/camunda/demo/process_payments/ProcessPaymentsApplication.java) with the completed code.
+:::
+
+Re-run the application in your terminal with `mvn spring-boot:run` to see the process run, and note the instance history in Operate.
+
+## Step 8: Deploy the process
+
+To deploy your process, take the following steps:
+
+1. Add `@Deployment(resources = "classpath:process-payments.bpmn")` to `ProcessPaymentsApplication.java`:
```

Comment:
```suggestion
1. Decorate the `ProcessPaymentsApplication` class with `@Deployment(resources = "classpath:process-payments.bpmn")` in `ProcessPaymentsApplication.java`:
```

precision -- they might not understand that the attribute gets applied to the class itself.